src/Kobo/System.hpp

```diff
@@ -71,3 +71,12 @@ KoboGetBacklightBrightness();
 
 void
 KoboSetBacklightBrightness(int percent);
+
+bool
+KoboCanChangeBacklightColour();
```

Comment:
Can this method throw exceptions?

Reply:
I don't see how it could throw an exception. It reads a string from the driver (which is noexcept) and calls atoi() which returns 0 in the case of invalid.

Reply:
I added noexcept